Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -480,14 +481,19 @@ protected override void GetChildItems(string path, bool recurse)
                             {
                                 GetChildItems(bucket.Name, true);
                             }
-                            catch (Exception e)
+                            catch (GoogleApiException e) when (e.HttpStatusCode == HttpStatusCode.Forbidden) { }
```

Comment:
Given that `catch (Exception) { }` is such a bad code smell, please leave a comment for why it is OK to silently swallow 403 errors, since it isn't immediately obvious. (i.e. it seems like a bug, right?)